### PR TITLE
Update Accents/Themes and add "Windows Accent"

### LINKED ISSUE #3294: Inherit Windows 8/10 system color for the UI color.
An interesting feature to consider implementing is to have HDT inherit the Windows 10 system color for the main UI color. 

image

#### Technical:

There's a value at the key  `Computer\HKEY_CURRENT_USER\Software\Microsoft\Windows\DWM`

DWORD `ColorizationColor` 

Stored as a decimal, turned hexadecimal and removing the first two characters  (eg `c400b7c3` -> `00b7c3`) gives the current dominate color the user has chosen.

This could be an option in settings to have it inherit Windows color, or it could be enabled by default to give Windows 10 the feeling of unification between apps (which is Microsoft's plan).

### PR DESCRIPTION
**Pull request by Zawodowiec1532**
Closes #3294 :)

### CODE REVIEW

File: Hearthstone Deck Tracker/Core.cs [hunk 1]
```diff
@@ -85,7 +85,8 @@ public static class Core
 			var newUser = ConfigManager.PreviousVersion == null;
 			LogConfigUpdater.Run().Forget();
 			LogConfigWatcher.Start();
-			Helper.UpdateAppTheme();
+			UITheme.InitializeTheme();
+			UITheme.ShowHDTLogo();
```
Comment (azeier) [inline Hearthstone Deck Tracker/Core.cs]: `ShowHDTLogo` doesn't seem to be named correctly. But that aside, why is this called separately here (a separate method at all) and not at the end of `InitializeTheme`?

File: Hearthstone Deck Tracker/UITheme.cs [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+			var accentName = "Windows Accent";
+			var fileName = Path.Combine(Config.Instance.ConfigDir, "WindowsAccent.xaml"); // Combine path to HDT folder + xaml file.
+
+			// Using FileStream to overwrite existing WindowsAccent file with new values.
+			using (FileStream s = new FileStream(fileName, FileMode.OpenOrCreate, FileAccess.ReadWrite))
+			using (var writer = System.Xml.XmlWriter.Create(s, new System.Xml.XmlWriterSettings { Indent = true }))
+			{
+				System.Windows.Markup.XamlWriter.Save(resourceDictionary, writer);
+				writer.Close();
+				s.Close();
```
Comment (azeier) [inline Hearthstone Deck Tracker/UITheme.cs]: Close calls are unnecessary, the using statements do that

File: Hearthstone Deck Tracker/UITheme.cs [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+			resourceDictionary.Add("MetroDataGrid.MouseOverHighlightBrush", new SolidColorBrush((Color)resourceDictionary["AccentColor3"]));
+			resourceDictionary.Add("MetroDataGrid.FocusBorderBrush", new SolidColorBrush((Color)resourceDictionary["AccentColor"]));
+			resourceDictionary.Add("MetroDataGrid.InactiveSelectionHighlightBrush", new SolidColorBrush((Color)resourceDictionary["AccentColor2"]));
+			resourceDictionary.Add("MetroDataGrid.InactiveSelectionHighlightTextBrush", new SolidColorBrush((Color)resourceDictionary["IdealForegroundColor"]));
+
+			var accentName = "Windows Accent";
+			var fileName = Path.Combine(Config.Instance.ConfigDir, "WindowsAccent.xaml"); // Combine path to HDT folder + xaml file.
+
+			// Using FileStream to overwrite existing WindowsAccent file with new values.
+			using (FileStream s = new FileStream(fileName, FileMode.OpenOrCreate, FileAccess.ReadWrite))
```
Comment (azeier) [inline Hearthstone Deck Tracker/UITheme.cs]: Should be var

**Comment by judge2020** [comment]
Based on https://superuser.com/questions/639921/windows-8-pro-version-info-in-registry and results at goo.gl/search/windows 8+product+name+registry the registry key check should work.

**Comment by azeier** [comment]
Merged as https://github.com/HearthSim/Hearthstone-Deck-Tracker/compare/184d6aaf3d0c...75743a815d20

**Pull request closed without merging**